<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReportVariantDialog</class>
 <widget class="QDialog" name="ReportVariantDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>438</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Report configuration for variant</string>
  </property>
  <property name="windowIcon">
   <iconset resource="GSvar.qrc">
    <normaloff>:/Icons/Report.png</normaloff>:/Icons/Report.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="0,1,0,0">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>inheritance:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>comp-het:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>de novo:</string>
     </property>
     <property name="buddy">
      <cstring>de_novo</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QComboBox" name="inheritance"/>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QLabel" name="variant">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>type:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>mosaic:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>comments:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QLabel" name="inheritance_hint">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="GSvar.qrc">:/Icons/Info.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="3">
    <widget class="QComboBox" name="type"/>
   </item>
   <item row="3" column="1" colspan="3">
    <widget class="QCheckBox" name="de_novo">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="3">
    <widget class="QCheckBox" name="mosaic">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="3">
    <widget class="QCheckBox" name="comp_het">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="6" column="1" colspan="3">
    <widget class="QPlainTextEdit" name="comments"/>
   </item>
   <item row="7" column="0" colspan="4">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="GSvar.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ReportVariantDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ReportVariantDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
